src/components/PopupDelete.js

```diff
@@ -0,0 +1,37 @@
+import { eventType, selectors } from '../constants/app-data.js';
+import Popup from './Popup.js';
+
+export default class PopupDelete extends Popup {
+	constructor(popupSelector) {
+		super({ popupSelector });
+		this._formElement = this._popupElement.querySelector(
+			selectors.modal.form
+		);
+	}
+
+	open(deleteHandler) {
+		this._deleteHandler = deleteHandler;
+		super.open();
+	}
+
+	close() {
+		this._formElement.removeEventListener(
```

Comment:
You don't need to remove this listener because you don't add `this._deleteHandler` as a submit handler